manifest.json

```diff
@@ -22,9 +22,12 @@
     "scripts": [
       "js/messaging.js",
       "vendor-overwrites/lz-string/LZString-2xspeedup.js",
+      "js/usercss.js",
       "background/storage.js",
       "js/prefs.js",
+      "js/script-loader.js",
       "background/background.js",
+      "vendor/node-semver/semver.js",
```

Comment:
This semver file is loaded dynamically in several places, does it need to be loaded in the background page?

Reply:
semver is used by `update.js`.